assets/js/components/ToggleableCaption.js

```diff
@@ -17,6 +18,13 @@ module.exports = class ToggleableCaption {
       return;
     }
 
+    if (this.thresholdWidth != null) {
```

Comment:
Use either `if (this.thresholdWidth)` if any truthy value will do, or if you're specifically wanting to check for the absence of `null`, use `if (this.thresholdWidth !== null)`